src/NuGet/BuildNuGets.csx

```diff
@@ -33,6 +34,17 @@ var BuildingReleaseNugets = IsReleaseVersion(BuildVersion);
 var NuspecDirPath = Path.Combine(SolutionRoot, "src/NuGet");
 var OutDir = Path.GetFullPath(Args[2]).TrimEnd('\\');
 
+var CommitSha = Args[3].Replace("<", "").Replace(">", "");
+var CommitIsDeveloperBuild = CommitSha == "<developer build>";
```

Comment:
@jaredpar - I just noticed this as part of resolving a merge conflict. Since you are stripping out the `<` and `>` chars, shouldn't the second line compare `CommitSha == "developer build"`?